docs/content.zh/docs/connectors/legacy-flink-cdc-sources/db2-cdc.md

```diff
@@ -32,9 +32,9 @@ describes how to setup the db2 CDC connector to run SQL queries against Db2 data
 
 ## Supported Databases
 
-| Connector             | Database                                           | Driver               |
-|-----------------------|----------------------------------------------------|----------------------|
-| [Db2-cdc](../db2-cdc) | <li> [Db2](https://www.ibm.com/products/db2): 11.5 | Db2 Driver: 11.5.0.0 |
+| Connector | Database                                           | Driver               |
+|-----------|----------------------------------------------------|----------------------|
+| Db2-cdc   | <li> [Db2](https://www.ibm.com/products/db2): 11.5 | Db2 Driver: 11.5.0.0 |
```

Comment:
What about using `{{ < ref > }}` here instead of removing the link?

Reply:
This just link to current page ,so I think there is no need to add this link.

Reply:
Thanks for the explanation! Yep we can just remove the link to the same page